src/components/tx/GasParams/index.tsx

```diff
@@ -51,19 +61,22 @@ const GasParams = ({ params, isExecution, isEIP1559, onEdit, gasLimitError }: Ga
   }
 
   return (
-    <Accordion elevation={0} onChange={onChangeExpand}>
-      <AccordionSummary>
+    <Accordion elevation={0} onChange={onChangeExpand} className={classnames({ [css.accordionSponsored]: willRelay })}>
+      <AccordionSummary className={css.accordionSummary}>
         {isExecution ? (
-          <Typography display="flex" alignItems="center" justifyContent="space-between" width={1}>
-            <span>Estimated fee </span>
-            {gasLimitError ? null : isLoading ? (
-              <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '7em' }} />
-            ) : (
-              <span>
-                {totalFee} {chain?.nativeCurrency.symbol}
-              </span>
-            )}
-          </Typography>
+          <>
+            <Typography display="flex" alignItems="center" justifyContent="space-between" width={1}>
+              <span>Estimated fee </span>
+              {gasLimitError ? null : isLoading ? (
+                <Skeleton variant="text" sx={{ display: 'inline-block', minWidth: '7em' }} />
+              ) : (
+                <span className={classnames({ [css.sponsoredFee]: willRelay })}>
+                  {totalFee} {chain?.nativeCurrency.symbol}
+                </span>
+              )}
+            </Typography>
+            {willRelay ? <div className={css.sponsoredChip}>Sponsored by Gnosis Chain</div> : null}
```

Comment:
Is `Sponsored by Gnosis Chain` something thats gonna be dynamic later on? Might be good to already extract it.